source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionSetRenderer.cs

```diff
@@ -138,10 +145,56 @@ public static void AddActions(Grid uiContainer, IList<AdaptiveAction> actions, A
             }
         }
 
+        public static void AddActionsToRoot(AdaptiveRenderContext context)
+        {
+            // Only handle Action show cards for the main card
+            if (context.CardDepth == 1)
+            {
+                if (context.CardRoot is Grid)
```

Comment:
>if (context.CardRoot is Grid) [](start = 16, length = 29)

you can do if (context.CardRoot is Grid variableName) and skip the casting in the line below